core/templates/dev/head/domain/skill/ConceptCardBackendApiService.ts

```diff
@@ -79,18 +79,18 @@ oppia.factory('ConceptCardBackendApiService', [
        * backend, it will store it in the cache to avoid requests from the
        * backend in further function calls.
        */
-      loadConceptCard: function(skillId) {
+      loadConceptCards: function(skillIds) {
         return $q(function(resolve, reject) {
-          if (_isCached(skillId)) {
+          if (_isCached(skillIds)) {
             if (resolve) {
-              resolve(angular.copy(_conceptCardCache[skillId]));
+              resolve(angular.copy(_conceptCardCache[skillIds]));
```

Comment:
I didn't understand the _conceptCardCache[skillIds] bit. the skillIds is a list no? Is that a key here?

Reply:
Yeah it's a list with multiple skill ids, and it's the key.

Reply:
This seems a little weird to me, why do you have a list of skill IDs as the key? Isn't a concept card linked to a single skill ID ?
Also having a list of variables as a key seems a little off to me - what if the order is different at time of creation and time of calling the map?

Reply:
Done.